skyfield/positionlib.py

```diff
@@ -91,7 +91,7 @@ def speed(self):
         """
         return Velocity(length_of(self.velocity.au_per_d))
 
-    def radec(self, epoch=None):
+    def radec(self, epoch=None, cirs=False):
```

Comment:
Sounds good. Although I have another suggestion I'll put in another comment.

Reply:
Thanks the the comment and for the suggestion that there's a different way forward here (making objects for coordinates, not just methods). But since I suspect that many Skyfield users are well served by the distinction between positions (objects) and coordinates (methods), let's move forward with the plan to make this a separate method. Since some folks might want the xyz coordinates, feel free to break this into two methods `cirs_xyz()` and `cirs_radec()` like ecliptic and galactic coordinates do.

I look forward to merging the result — thanks again!